src/topology/algebra/module/weak_dual.lean

```diff
@@ -173,7 +173,7 @@ weak_bilin (top_dual_pairing 𝕜 E)
 instance : inhabited (weak_dual 𝕜 E) :=
 by {dunfold weak_dual, dunfold weak_bilin, apply_instance}
 
-instance fun_like_weak_dual : fun_like (weak_dual 𝕜 E) E (λ _, 𝕜) :=
+instance add_monoid_hom_class_weak_dual : add_monoid_hom_class (weak_dual 𝕜 E) E 𝕜 :=
 by {dunfold weak_dual, dunfold weak_bilin, apply_instance}
```

Comment:
```suggestion
instance add_monoid_hom_class_weak_dual : add_monoid_hom_class (weak_dual 𝕜 E) E 𝕜 :=
continuous_linear_map.add_monoid_hom_class
```
I don't want to change it in #12623 to avoid merging issues.

Reply:
Ah, thanks!